mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/profile/ProfileStateScreen.kt

```diff
@@ -0,0 +1,174 @@
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.CheckBox
+import androidx.compose.material.icons.filled.Edit
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.ui.features.ProfileColor
+import ch.epfl.sdp.mobile.ui.features.social.ChessMatch
+import ch.epfl.sdp.mobile.ui.features.social.MatchResult
+import ch.epfl.sdp.mobile.ui.i18n.LocalLocalizedStrings
+
+val state =
+    object : ProfileState {
+      override val email: String = "badrtaddist1@gmail.com"
+      override val numberOfGames: Int = 10
+      override val numberOfPuzzles: Int = 10
+      override val matches: List<ChessMatch> =
+          List(20) { ChessMatch("Konor($it)", MatchResult.WIN, MatchResult.Reason.CHECKMATE, 27) }
+
+      override fun onSettingsClick() {}
+      override fun onEditClick() {}
+      override val backgroundColor: ProfileColor = ProfileColor.Pink
+      override val name: String = "Badr Taddist"
+      override val emoji: String = "😊"
+    }
+
+@Preview
+@Composable
+fun preview() {
+  //  SettingsButton(state::onSettingsClick)
+  // ProfileHeader(state = state)
+  //  GamesInfo(numPastGames = 42, numPuzzles = 10)
+  ProfileScreen(state = state)
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun ProfileScreen(state: ProfileState, modifier: Modifier = Modifier) {
+  val strings = LocalLocalizedStrings.current
+  LazyColumn(
+      verticalArrangement = Arrangement.Top,
+      horizontalAlignment = Alignment.CenterHorizontally,
+      modifier = modifier) {
+    item { ProfileHeader(state) }
+    stickyHeader {
+      GamesInfo(numPastGames = state.numberOfGames, numPuzzles = state.numberOfPuzzles)
+    }
+    items(state.matches) { match ->
+      val title = strings.profileMatchTitle(match.adv)
+      val subtitle = strings.profileMatchInfo(match.matchResult, match.cause, match.numberOfMoves)
+      Match(title, subtitle, modifier = modifier)
+    }
+  }
+}
+
+@Composable
+fun ProfileHeader(state: ProfileState, modifier: Modifier = Modifier) {
+  Column(
+      modifier = modifier,
+      horizontalAlignment = Alignment.CenterHorizontally,
+  ) {
+    Spacer(Modifier.height(16.dp))
+    ProfilePicture(state)
+    Spacer(Modifier.height(16.dp))
+    Text(state.name, style = MaterialTheme.typography.h5)
+    Text(state.email, style = MaterialTheme.typography.subtitle2)
+    Spacer(Modifier.height(16.dp))
+    SettingsButton(onClick = state::onSettingsClick)
+    Spacer(Modifier.height(16.dp))
+  }

```

Comment:
I suggest using `Arrangement.spacedBy` with a value of `16.dp`, and grouping some of the items in some inner `Column` (such as the two text), to avoid repeating `Spacer(Modifier.height(16.dp))` too many times.